src/User32.Desktop/User32.cs

```diff
@@ -1,11 +1,15 @@
 ﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 
+#pragma warning disable SA1625 // Element documentation must not be copied and pasted
```

Comment:
In was already disabled everywhere that triggered it, so I disabled it globally for this file.
It seem to me that it will always trigger uselessly, lot of API have shared documentation.